src/signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/DetailPanel.tsx

```diff
@@ -158,6 +186,39 @@ const DetailPanel: FC<DetailPanelProps> = ({ language = {} }) => {
     setOptionsList(null)
   }, [removeItem])
 
+  useEffect(() => {
+    if (!selectionOnMap || !selection?.coordinates || !category || !subcategory)
+      return
+
+    const searchParams = new URLSearchParams({
+      maincategory_slug: category,
+      category_slug: subcategory,
+      lat: selection.coordinates.lat.toString(),
+      lon: selection.coordinates.lng.toString(),
+      group_by: 'category',
+    })
+
+    get(`${configuration.GEOGRAPHY_PUBLIC_ENDPOINT}?${searchParams.toString()}`)
+  }, [get, selectionOnMap, selection, category, subcategory])
+
+  useEffect(() => {
+    setSelectionIncident({})
+
+    if (selectionNearby) {
+      setSelectionIncident({
+        categoryName: selection?.label,
+        createdAt: selection?.description,
+      })
+    }
+
+    if (selectionOnMap && data?.features) {
```

Comment:
FeatureCollection does seem to make features required, so optional chain could be removed?!

Reply:
If I remove it, the build fails